MacrobenchmarkSample/com.example.macrobenchmark-benchmarkData.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
did you mean to check this in? it will get stale very quickly when any benchmarks change

Reply:
ACK